modules/caddyhttp/headers/headers.go

```diff
@@ -132,7 +132,7 @@ type HeaderOps struct {
 }
 
 // Provision sets up the header operations.
-func (ops *HeaderOps) Provision(_ caddy.Context) error {
+func (ops *HeaderOps) Provision(ctx caddy.Context) error {
```

Comment:
This variable isn't used, so I'm not sure why it needs a name. Naming it falsely implies that we're using it, whereas `_` makes it clear we're satisfying an interface IMO.

Reply:
OK, I can change that setting, and that'd get applied universally.

Reply:
Universally for context args, or for every arg?

Because I think it could make sense for some args.

If it can't be configured in that way, I don't think that linter is helpful for us.